docker/build-aws.sh

```diff
@@ -23,31 +23,31 @@ BUILD_PROJECTS=()
 
 while [[ "$1" =~ ^- ]]; do case $1 in
   --build-all )
-    BUILD_PROJECTS=(libra-validator libra-cluster-test libra-init libra-mint libra-safety-rules libra-tools)
+    BUILD_PROJECTS=(diem-validator diem-cluster-test diem-init diem-mint diem-safety-rules diem-tools)
```

Comment:
We need to land a corresponding change in libra-ops first so that the LBT would work. 

I will make that change after this msg.

Reply:
Sounds good! Thanks so much...testing this all out is a real pain lol. 

Also, I just rebased, so going to try a canary

Reply:
The change has been applied.  Give /canary LBT a shot.